Tropos/Models/Precipitation.swift

```diff
@@ -0,0 +1,23 @@
+import Foundation
+
+@objc enum PrecipitationChance: Int {
+    case None, Slight, Good
+}
+
+@objc class Precipitation: NSObject {
+    let type: String
+    let probability: Float
+    
+    var chance: PrecipitationChance {
+        switch probability {
+        case probability where probability > 30.0: return .Good
```

Comment:
Thoughts on these being `case _ where` since we aren't using the probability here?